src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+/*Reset default margins*/
+html,
+body,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-active-color: #00acdc;
+  --link-color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
+  height: auto;
+  margin-right: 50px;
```

Comment:
let's give horizontal paddings to header instead

Reply:
✅ Done